manage.py

```diff
@@ -31,6 +31,19 @@ def sighandler(signum, frame):
     # http://blog.lotech.org/fix-djangos-runserver-when-run-under-docker-or-pycharm.html
     signal.signal(signal.SIGTERM, sighandler)
 
+    try:
+        import django
+    except ImportError:
+        if os.environ.get('virtual_env') is None:
+            print('\033[36;1mINFO: No virtual_env detected,',
+            'usually contributors use virtual environment `zdsenv`\033[00;0m')
+        elif os.path.normpath(os.environ.get('virtual_env')) != os.path.normpath('./zdsenv'):
+            print('\033[36;1mINFO: The current virtual_env is `%s`,',
+            'usually contributors use `./zdsenv`\033[00;0m' % os.environ.get('virtual_env'))
+
+        print("\033[31;1mModuleNotFoundError: No module named 'django' found\033[00;0m")
```

Comment:
Je ne suis VRAIMENT pas fan d'avoir un print qui va ressembler à une exception sans en être une. À debug c'est un enfer.

Reply:
C'est en cyan et il y a écrit "INFO" devant, je ne vois pas le soucis

Reply:
en effet "your environment does not have django, please run pip install -r requirements-dev.txt" serait meilleur

Reply:
Ahh tu parles de l'erreur ligne 44 ?

Reply:
Exact, c'est bien cette ligne que j'ai commenté.

Reply:
Je pense que tu peux supprimer le `ModuleNotFoundError` qui pour moi ne sert à rien (c'est d'ailleurs ce que j'entendais dans mon premier message). C'est la syntaxe d'une exception or c'est seulement un print. Le message se suffit à lui-même, pas besoin de faire croire qu'il y a une exception derrière alors que ce n'est pas le cas.